tests/components/pglab/test_config_flow.py

```diff
@@ -0,0 +1,113 @@
+"""Test the PG LAB Electronics config flow."""
+
+from homeassistant import config_entries
+from homeassistant.components.pglab.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.service_info.mqtt import MqttServiceInfo
+
+from tests.common import MockConfigEntry
+from tests.typing import MqttMockHAClient
+
+
+async def test_mqtt_abort_if_existing_entry(
```

Comment:
This test can be parametrized to remove the `test_user_single_instance` test

Reply:
Good point.